packages/odyssey/src/scss/components/_text-input.scss

```diff
@@ -14,6 +14,26 @@
 
 .ods-text-input {
   @include input-baseline;
+
+  &[type='search'] {
+    padding-left: ms(1) + $spacing-s + $spacing-xs;
+    background-image: get-icon('search', $text-body);
```

Comment:
We've tried to address this by using an SR-accessible label even when the indicator is hidden. In End User, was it an issue even with alternate labeling?

@arnoldsandoval-okta do you think this falls under a piece of UI that non-SR-but-alternate-mode users might need to see? We'll run into similar issues for things like checkbox checks and dropdown carets.

Reply:
This is good feedback. @santhoshbalakrishnan can you expand on what you mean by "certain modes", I'm interested in further exploring the implications of this and coming up with some hard rules!

Reply:
Some context:

<img width="912" alt="Screen Shot 2020-10-01 at 3 23 28 PM" src="https://user-images.githubusercontent.com/24683447/94869409-2d33aa00-03fa-11eb-8474-b6c6ccb793a8.png">
<img width="913" alt="Screen Shot 2020-10-01 at 3 23 37 PM" src="https://user-images.githubusercontent.com/24683447/94869419-2f960400-03fa-11eb-8476-1507db2ae047.png">

Reply:
Reading back, I think we maybe okay here since we have `Search` as a text at some place in the input field.

Main part was on some high contrast modes/turn off colors, the background images can be turned off.